content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -195,7 +195,7 @@ custom:
         -- find . -name "libquery_engine-*" -not -name "libquery_engine-rhel-openssl-*" | xargs rm
 ```
 
-If you are deploying to [Lambda functions with ARM64 architecture](#lambda-functions-with-arm64-architectures) you should update the find command to the following:
+If you are deploying to [Lambda functions with ARM64 architecture](#lambda-functions-with-arm64-architectures) you should update the `find` command to the following:
 
 ```sh
```

Comment:
```suggestion
```terminal
```

Reply:
Keeping this was intentional since it's not a command to be run in the terminal, it's suggesting how you change the `find` command on line 195

Maybe this will work? 
```suggestion
```
```

Reply:
Ah, sorry 🤦 

In that case, I actually think it's most clear if we repeat the entire file. I'll make a separate suggestion.